app/brave_generated_resources.grd

```diff
@@ -963,6 +963,18 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
 
       <!-- Brave VPN -->
       <if expr="enable_brave_vpn">
+        <message name="IDS_SETTINGS_VPN_PAGE_TITLE" desc="Title of the vpn settings page">
+          Brave VPN
+        </message>
+        <message name="IDS_SETTINGS_VPN_PAGE_USE_WIREGUARD_TITLE" desc="Title of the Wireguard option on the vpn settings page">
+          Use Wireguard protocol in Brave VPN
```

Comment:
```suggestion
          Use WireGuard protocol in Brave VPN
```

Reply:
We might just scan over places in general and make sure it's stylized (to users) as `WireGuard`

Reply:
done